variables.tf

```diff
@@ -212,7 +212,7 @@ variable "nfs_disk_size" {
   default = 250
 }
 
-# container registry
+# container registry - WIP
```

Comment:
Should this really be Work In Progress or TODO in the future?

Reply:
I can change it from `WIP`->`TODO`, just wanted to add a small note next to the variable for users who dig into the code to see that this var is not fully complete at this time.

Reply:
done.